lisa/target.py

```diff
@@ -142,6 +142,7 @@ class TargetConf(SimpleMultiSrcConf, HideExekallID):
         KeyDesc('device', 'ADB device. Takes precedence over "host"', [str, None]),
         KeyDesc('keyfile', 'SSH private key file', [str, None]),
         KeyDesc('strict-host-check', 'Equivalent to StrictHostKeyChecking option of OpenSSH', [bool, None]),
+        KeyDesc('use-scp', 'Use SCP as method of file transfer instead of the default SFTP', [bool, None]),
```

Comment:
Would be best exposed under the "devlib" key (since it's more a devlib-specific workaround for unusual setup) and as "file-xfer" key, with either "scp" or "sftp" string values, so that we can reuse it if any other connection type have alternate backends.

In the doc string, it's a good idea to mention that it only applies to the "linux" kind (i.e. it's ignored for android).

In `Target.__init__` it will map automatically to an `devlib_file_xfer=None` parameter (just like for `devlib_excluded_modules`).

Reply:
Actually I don't remember if it's automatic, you may have to "manually" map it in `Target.INIT_KWARGS_KEY_MAP`

Reply:
Updated.